src/github.com/stellar/go-horizon/actions_root.go

```diff
@@ -1,29 +1,34 @@
 package horizon
 
 import (
-	"net/http"
-
 	"github.com/jagregory/halgo"
 	"github.com/stellar/go-horizon/render/hal"
 )
 
 // RootResource is the initial map of links into the api.
 type RootResource struct {
+  HorizonVersion string
```

Comment:
In the rest of the codebase, we put embedded structs at the top of the definition.  Could you make this struct consistent with the others?